test/raft/raft_address_map_test.cc

```diff
@@ -188,24 +160,26 @@ SEASTAR_THREAD_TEST_CASE(test_raft_address_map_replication) {
     using seastar::manual_clock;
 
     {
-        sharded<raft_address_map<manual_clock>> m_svc;
+        sharded<raft_address_map_t<manual_clock>> m_svc;
         m_svc.start().get();
         auto stop_map = defer([&m_svc] { m_svc.stop().get(); });
         auto& m = m_svc.local();
 
         // Replicate non-expiring entry, ensure it doesn't expire on the other shard
```

Comment:
s/entry/flag; we don't replicate entries now, we only replicate the (non)expiration flag.
I'd reword to something like:
```
// Add entry on both shards, replicate non-expiration flag, ensure it doesn't expire on the other shard
```